mission_data/templates/temp_test.yml

```diff
@@ -1,41 +0,0 @@
-apiVersion: influxdata.com/v2alpha1
```

Comment:
This file should not be deleted, but can you modify it so that it still works

Reply:
I have doubts about this file. I will need your help in making changes.